provision.py

```diff
@@ -158,7 +158,7 @@ def setup_virtualenv(target_venv_path, requirements_file, virtualenv_args=None):
     # type: (str, str, List[str]) -> None
 
     # Check if a cached version already exists
-    output = subprocess.check_output(['sha1sum', requirements_file])
+    output = subprocess.check_output([os.path.join('tools', 'hash_reqs.py'), requirements_file])
```

Comment:
Yeah, I think what's happening is that GitHub is sorting by author date, not committer date, which is just wrong.